src/Navbar/Navbar.component.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+export type Props = {
+  /**
+   * A color defined in theme colors or a CSS color code
+   * or shorthand string for setting element background
+   *
+   * @default 'white'
+   **/
+  background?: string;
+  /**
+   * The shorthand string for setting element border-top
+   *
+   * @default ''
+   **/
+  borderTop?: string;
+  /**
+   * The shorthand string for setting element border-bottom
+   *
+   * @default ''
+   **/
+  borderBottom?: string;
+  /**
+   * Adjust Navbar height.
+   *
+   * @default '80px'
+   **/
+  height?: string;
+  /**
+   * From theme provider
+   *
+   * @default defaultTheme
+   **/
+  theme?: any;
+};
+
+const SNavbar = styled.div`
+  font-family: ${(props: Props) => props.theme.typography.fontFamily};
+  font-size: ${(props: Props) => props.theme.typography.fontSize};
+  color: ${(props: Props) => props.theme.typography.color};
```

Comment:
let's have color defined int he navbar section of the theme

Reply:
@ben-dalton I have defined navbar color in the navbar section of the theme